app/models/audio_file_template.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+class AudioFileTemplate < BaseModel
```

Comment:
This is turning out to be bigger than I wanted - I'm going to fix the template related policies, then do a 2nd pr with the clean up for the rest of the models